docs/authentication.rst

```diff
@@ -62,11 +62,11 @@ may be useful:
     sa = sgtk.authentication.ShotgunAuthenticator()
 
     # Use the authenticator to create a user object. This object
-    # identifies a Shotgun user or script and also wraps around
-    # a Shotgun API instance which is associated with that user.
+    # identifies a ShotGrid user or script and also wraps around
+    # a ShotGrid API instance which is associated with that user.
     user = sa.create_script_user(api_script="myscript",
                                  api_key="xxxxx",
-                                 host="https://myhost.shotgunstudio.com")
+                                 host="https://myhost.shotgridstudio.com")
```

Comment:
Is this correct? Do we know this URL already?

Reply:
Actually, it's supposed to be `.shotgrid.autodesk.com`